modules/apps/configuration-admin/configuration-admin-api/src/main/java/com/liferay/configuration/admin/configuration/InstanceSettingsClientExtensionsConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.configuration.admin.configuration;
```

Comment:
Wouldn't this make more sense to be in portal-settings-web? What do you guys think?
@algobob @thiago-buarqque

Reply:
Agreed, missed that one. We won't be making this configuration available to read anywhere else besides the configuration-admin-web module, which contains the DynamicInclude

Reply:
I think portal-settins-web makes more sense for all this since it's related to instance settings specifically. configuration-admin-web looks more related to the entire configuration framework, rather than instance settings specifically. What do you guys think?

Reply:
Ops, it's portal-settings-web, sorry I had configuration admin in my mind cause of the class package.

The instance appearance configuration it's in portal-settings-web `AppearanceEditCompanyPortalSettingsConfigurationScreenWrapper`. So, I agree with you.